modules/installation-configuration-parameters.adoc

```diff
@@ -1727,6 +1735,50 @@ Additional Azure configuration parameters are described in the following table:
 |Defines the type of disk.
 |`premium_LRS` or `standardSSD_LRS`. The default is `premium_LRS`.
 
+|`controlPlane.platform.azure.type`
+|Defines the azure instance type.
```

Comment:
How about rewarding to Defines the azure instance type for control plane machines ?